bioio/tests/writers/test_timeseries_writer.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Callable, Tuple
+
+import bioio_base as biob
+import imageio
+import numpy as np
+import pytest
+
+from bioio.writers.timeseries_writer import TimeseriesWriter
+from bioio.writers.two_d_writer import TwoDWriter
+
+from ..conftest import array_constructor, get_resource_write_full_path
+
+
+@array_constructor
+@pytest.mark.parametrize(
+    "write_shape, write_dim_order, read_shape",
+    [
+        # TODO: Failing currently, needs work,
+        # see https://github.com/bioio-devs/bioio/issues/10
```

Comment:
These two commented out tests pass on `aicsimageio` (technically) see the issue for why this was commented out.